dataset.py

```diff
@@ -9,27 +11,43 @@
 class Dataset():
     """The Google Analytics dataset."""
 
-    def __init__(self):
+    def __init__(self, debug=False):
         """Load the data from disk."""
```

Comment:
Add an "args" section to the docstring. Here's an example: https://gist.github.com/dsaiztc/9b4c15c6dfef08a957c1#file-docstring_google-py-L64